docs/root/intro/version_history.rst

```diff
@@ -4,17 +4,123 @@ Version history
+  based on the :ref:`percentage of healthy servers in upstream clusters
+  <envoy_api_field_config.filter.http.health_check.v2.HealthCheck.cluster_min_healthy_percentages>`.
 * health check: added setting for :ref:`no-traffic
   interval<envoy_api_field_core.HealthCheck.no_traffic_interval>`.
-* tracing: when using the zipkin tracer, it is now possible for clients to specify the sampling decision (using
-  the :ref:`x-b3-sampled <config_http_conn_man_headers_x-b3-sampled>` header) and have the decision
-  propagated through to subsequently invoked services.
-* tracing: when using the zipkin tracer, it is no longer necessary to propagate the
+* http : added idle timeout for :ref:`upstream http connections
+  <envoy_api_field_core.HttpProtocolOptions.idle_timeout>`.
+* http: added support for :ref:`proxying 100-Continue responses
+  <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.proxy_100_continue>`.
+* http: added the ability to pass a URL encoded PEM encoded peer certificate in the
+  :ref:`config_http_conn_man_headers_x-forwarded-client-cert` header.
+* http: added support for trusting additional hops in the
+  :ref:`config_http_conn_man_headers_x-forwarded-for` request header.
+* http: added support for :ref:`incoming HTTP/1.0
+  <envoy_api_field_core.Http1ProtocolOptions.accept_http_10>`.
+* hot restart: added SIGTERM propagation to children to :ref:`hot-restarter.py
+  <operations_hot_restarter>`, which enables using it as a parent of containers.
+* ip tagging: added :ref:`HTTP IP Tagging filter<config_http_filters_ip_tagging>`.
+* listeners: added support for :ref:`listening for both IPv4 and IPv6
+  <envoy_api_field_core.SocketAddress.ipv4_compat>` when binding to ::.
+* listeners: added support for listening on :ref:`UNIX domain sockets
+  <envoy_api_field_core.Address.pipe>`.
+* listeners: added support for :ref:`abstract unix domain sockets <envoy_api_msg_core.Pipe>` on
+  Linux. The abstract namespace can be used by prepending '@' to a socket path.
+* listeners: added :ref:`transparent <envoy_api_field_Listener.transparent>` option.
+* load balancer: added cluster configuration for :ref:`healthy panic threshold
+  <envoy_api_field_Cluster.CommonLbConfig.healthy_panic_threshold>` percentage.
+* load balancer: added :ref:`Maglev <arch_overview_load_balancing_types_maglev>` consistent hash
+  load balancer.
+* load balancer: added support for
+  :ref:`LocalityLbEndpoints<envoy_api_msg_endpoint.LocalityLbEndpoints>` priorities.
+* lua: added headers :ref:`replace() <config_http_filters_lua_header_wrapper>` API.
+* redis: added local `PING` support to the :ref:`Redis filter <arch_overview_redis>`.
+* redis: added `GEORADIUS_RO` and `GEORADIUSBYMEMBER_RO` to the :ref:`Redis command splitter
+  <arch_overview_redis>` whitelist.
+* router: added DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT, DOWNSTREAM_LOCAL_ADDRESS,
```

Comment:
Is it worth merging this with the access_log entry up top?

Reply:
I didn't do that because one is about headers, and one is about access logs, but I could go either way. WDYT?

Reply:
Per offline convo going to keep it as is.

Reply:
Per slack, I'm fine leaving it.